pages/settings/index.js

```diff
@@ -33,7 +34,14 @@ export default function SettingsPage({
           </StyledDeletionWarning>
           <StyledCategoryName>Category: {categoryToDelete}</StyledCategoryName>
           <StyledConfirmActionContainer>
-            <StyledCancelButton type="button">Cancel</StyledCancelButton>
+            <StyledCancelButton
+              type="button"
+              onClick={() => {
+                handleCancelDeleteCategory();
+              }}
```

Comment:
this can be a bit shorter if you like, but entirely optional:
```suggestion
onClick={handleCancelDeleteCategory}
```